docs/EthicsOfBlockchainGovernance.md

```diff
@@ -2,19 +2,19 @@
 
 ## Principles
 
-1. One `de facto` governs some thing if one can create and destroy that thing.
+1. One `de facto` governs something if one can create and destroy that thing.
```

Comment:
```suggestion
1. One `de facto` governs some defined thing if one can create and destroy that thing.
```

Reply:
The accent I wanted to convey is subtle: "some thing" means defined and "something" means regardless of choice.